automated/deployment/lava_prog_device.sh

```diff
@@ -22,11 +22,17 @@ TEST_NAME="image_deployment"
 # find image and extract
 cd /lava-lxc || exit
 
-ARCHIVE=$(find . -iname "*tar*")
-tar xzvf "$ARCHIVE"
-rm "$ARCHIVE"
 
-IMAGE=$(find . -iname "*.img")
+if ! command -v unxz &> /dev/null; then
+    echo "xz not intalled. It will now be installed."
+    apt-get -q update
+    apt-get install -qy xz-utils
+fi
+
+ARCHIVE=$(find . -iname "*xz*")
+xz --decompress "$ARCHIVE"
+
+IMAGE=$(find . -maxdepth 1 -iname "*.xz" -type f)
```

Comment:
This is the line where the fix for line 32 seems to have ended up. An `*.xz´ file shouldn't exist at this point anymore anyway as the `--keep` flag hasn't been passed.
Revert this line and change line 32 instead.

Reply:
@tboehler1 
Sorry, I modified the wrong line...

With the latest changes, it works [https://lava.revpi.kunbus.com/scheduler/job/1632](https://lava.revpi.kunbus.com/scheduler/job/1632)